vcd_cli/org.py

```diff
@@ -139,3 +143,37 @@ def create(ctx, name, full_name):
         stdout('Org \'%s\' is successfully created.' % result.get('name'), ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@org.command(short_help='delete an organization')
+@click.pass_context
+@click.argument('name',
+                metavar='<name>',
+                required=True)
+@click.option('-r',
+              '--recursive',
+              is_flag=True,
+              help='removes the Org and any objects it contains that are in a '
+                   'state that normally allows removal')
+@click.option('-f',
+              '--force',
+              is_flag=True,
+              help='pass this option along with --recursive to remove an Org'
+                   ' and any objects it contains, regardless of their state')
+@click.option('-y',
+              '--yes',
+              is_flag=True,
+              callback=abort_if_false,
+              expose_value=False,
+              prompt='Are you sure you want to delete the org?')
+def delete(ctx, name, recursive, force):
+    try:
+        client = ctx.obj['client']
+        system = System(client)
```

Comment:
Does this internally check if logged-in-user is SysAdmin?

Reply:
We do not check it. we allow it to fail in the VCD.